package.json

```diff
@@ -19,6 +19,7 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
+    "async-retry": "^1.3.3",
```

Comment:
se for usar só para testes então pode virar uma dependência de desenvolvimento

Reply:
@rayllandersotero por volta do minuto `4:50` da aula [Estabilizar "npm test" (Orquestrador)](https://curso.dev/web/estabilizar-npm-test-orchestrator) o Deschamps cita que o `async-retry` poderá ser utilizado em outras partes do código como temos no TabNews. :+1:

Como exemplo veja o componente `infra/database`:

https://github.com/filipedeschamps/tabnews.com.br/blob/4d62fc6d87dcbab578dd26349800d3475b003a88/infra/database.js#L58-L64